rpc/cty_encode.go

```diff
@@ -4,10 +4,16 @@
 package rpc
 
 import (
+	"encoding/gob"
+
 	"github.com/zclconf/go-cty/cty"
 	"github.com/zclconf/go-cty/cty/json"
 )
 
+// Test that cty types implement the gob.GobEncoder interface.
+// Support for encoding/gob was removed in github.com/zclconf/go-cty@v1.11.0.
+var _ gob.GobEncoder = cty.Value{}
```

Comment:
Is it possible to put this inside an `init` method so you can check the type and pop an error message of our choice to include the related issue link? 🤔